keyless/pepper/common/src/vuf/bls12381_g1_bls.rs

```diff
@@ -14,11 +18,46 @@ use ark_std::{
     rand::{CryptoRng, RngCore},
     UniformRand,
 };
+use blstrs::{self, Compress};
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use serde_big_array::BigArray;
 use std::ops::Mul;
 
 pub struct Bls12381G1Bls {}
 
-pub static DST: &[u8] = b"APTOS_PEPPER_SERVICE_BLS12381_VUF_DST";
+pub static DST: &[u8] = b"APTOS_PEPPER_BLS12381_VUF_DST";
+
+pub static PINKAS_DST: &[u8] = b"APTOS_PINKAS_PEPPER_DST";
+
+pub static PINKAS_SECRET_KEY_BASE_SEED: &[u8] = b"APTOS_PINKAS_PEPPER_SECRET_KEY_BASE_SEED";
```

Comment:
@alinush

Reply:
I see. You can rename the variable, but leave the string contents like this.